composer.json

```diff
@@ -15,8 +15,10 @@
       "TenUpScaffold\\": "includes/classes/"
     }
   },
+  "minimum-stability": "dev",
+  "prefer-stable": true,
   "require-dev": {
-    "10up/wp_mock": "dev-dev",
+    "10up/wpacceptance": "dev-master",
```

Comment:
Can you update the readme with instructions on how to run these tests?

Reply:
Added in 290efd9, please proof 😃